_quarto.yml

```diff
@@ -39,9 +39,12 @@ book:
       chapters:
       - epipredict.qmd
       - forecast-framework.qmd
-      - flatline-forecaster.qmd
-      - tidymodels-intro.qmd
       - tidymodels-regression.qmd
```

Comment:
Is this actually the right order? Based on the titles, I'd suspect that "Intro..." would come before the "Regression..." parts.

Reply:
My thinking here was that this ordering was ok because the regression stuff gets used in the tidymodels framework & both more general intros should come before we get into the package-specific stuff, but there are definitely other ways to organize these chapters.